Fördermitglieder (#3)
chaos-jetzt/verein: satzung.md

### § 3b Beitritt

1. Der Aufnahmeantrag erfolgt in Textform gegenüber dem Vorstand.

> 2. Die Fördermitgliedschaft steht grundätzlich jeder natürlichen oder juristischen Person sowie Anstalt oder Körperschaft des öffentlichen Rechts offen.

Reviewer: ```suggestion
2. Die Fördermitgliedschaft steht grundätzlich jeder natürlichen Person offen.
```

    2. Tod von natürlichen Personen oder 
    3. Auflösung und Erlöschen von juristischen Personen, Handelsgesellschaften, nicht rechtsfähigen Vereinen sowie Anstalten und Körperschaften des öffentlichen Rechts oder
    4. Ausschluss.
2. Der Austritt wird durch Willenserklärung in Textform gegenüber dem Vorstand vollzogen. Der Austritt ist jederzeit mit einer Frist von vier Wochen möglich.
3. Die Beitragspflicht für das laufende Beitragsjahr bleibt unberührt. Eine Erstattung bereits vereinnahmter Beiträge erfolgt nicht.

> 4. Vollendet ein ordentliches Mitglied das siebenundzwanzigste Lebensjahr, so wird seine Mitgliedschaft automatisch zu einer Alumni-Mitgliedschaft, die in allen Rechten und Pflichten einer Fördermitgliedschaft gleichgestellt ist.

Reviewer 2: ```suggestion
4. Vollendet ein ordentliches Mitglied das siebenundzwanzigste Lebensjahr, so wird seine Mitgliedschaft automatisch zu einer Alumni-Mitgliedschaft, die in allen Rechten einer Fördermitgliedschaft gleichgestellt ist, mit den Pflichten einer ordentlichen Mitgliedschaft.
```
Ich bin sehr dagegen automatisch die Beiträge zu erhöhen, ohne dass die Mitglieder darum bitten.

Reviewer 3: Ich sehe die automatische Erhöhung ebenfalls kritisch.

Ich schlage vor, dass wir eine Möglichkeit schaffen das bestehende Mitglieder die Mitgliedschaft auf Antrag umwandeln können (was z.B. Relevant wäre falls wir eine Aufnahmegebühr hätten), und ohne dass der Vorstand den Mitgliedschaftsantrag erneut entscheiden muss bzw. ablehnen kann.

Author: Ich finde @e1mo s Idee da sehr gut.
Zur Umsetzung schlage ich follgendes vor:

```suggestion
4. Vollendet ein ordentliches Mitglied das siebenundzwanzigste Lebensjahr, endet seine ordentliche Mitgliedschaft mit den verbundenen Rechten und Pflichten. Das ehemalige Mitglied kann daraufhin einen Antrag auf Alumni-Mitgliedschaft stellen.
4.1. Eine Alumni-Mitgliedschaft kommt in ihren Rechten denen einer Fördermitgliedschaft gleich, in ihren Pflichten, insbesondere der Beitragspflicht, jedoch einem ordentlichen Mitglied. Eine erneute Aufnahmegebür fällt hierbei keinesfalls an.
```

### § 6e Stimmrecht

1. Jedes ordentliche Mitglied hat eine Stimme.

> 2. Das Stimmrecht kann auf andere stimmberechtigte Mitglieder übertragen werden. Das bevollmächtigte Mitglied muss eine schriftliche Stimmrechtsvollmacht vorlegen. Kein stimmberechtigtes Mitglied darf mehr als zwei Stimmrechtsvollmachten auf sich vereinigen.
> 3. Fördermitglieder sind nicht stimmberechtigt.

Reviewer 2: ```suggestion
    1.  Das Stimmrecht kann auf andere stimmberechtigte Mitglieder übertragen werden. Das bevollmächtigte Mitglied muss eine schriftliche Stimmrechtsvollmacht vorlegen. Kein stimmberechtigtes Mitglied darf mehr als zwei Stimmrechtsvollmachten auf sich vereinigen.
2. Fördermitglieder sind nicht stimmberechtigt.
```

Author: Der Neue Text würde vorsehen:

### § 6e Stimmrecht
   1.  Das Stimmrecht kann auf andere stimmberechtigte Mitglieder übertragen werden. Das bevollmächtigte Mitglied muss eine schriftliche Stimmrechtsvollmacht vorlegen. Kein stimmberechtigtes Mitglied darf mehr als zwei Stimmrechtsvollmachten auf sich vereinigen.
2. Fördermitglieder sind nicht stimmberechtigt.

Fehlen würde also das Stimmrecht für ordentliche Mitglieder.
Der Sinn dieser Änderung ergibt sich mir nicht.

### § 7a Besetzung des Vorstandes

1. Die Amtsdauer der Vorstandsmitglieder beträgt zwei Jahre. Wiederwahl ist zulässig. 
2. Der Vorstand bleibt bis zur Neuwahl im Amt, damit auch nach Ablauf der Amtsdauer eine ordnungsgemäße gesetzliche Vertretung gesichert ist.
3. Der Vorstand besteht aus natürlichen Personen, den Vorstandsmitgliedern. Die Bekleidung mehrer Ämter durch eine Person ist zulässig, sofern dies auf Beschluss der Mitgliederversammlung erfolgt.
4. Jedes Vorstandsmitglied besitzt in Abstimmungen und Beschlussfassungen des Vorstands eine Stimme, unbeschadet der Anzahl der ausgeübten Ämter.
5. Scheidet ein Vorstandsmitglied während seiner Amtszeit aus, wählt der verbleibende Vorstand übergangsweise bis zur nächsten Mitgliederversammlung einen Ersatz. Sind zwei oder mehr der durch die Mitgliederversammlung gewählten Vorstandsmitglieder dauerhaft an der Ausübung ihres Amtes gehindert, so sind unverzüglich Nachwahlen anzuberaumen.
6. Die Abwahl eines Vorstandsmitglieds ist nur möglich, sofern zeitgleich eine Neubesetzung des freigewordenen Amtes gemäß dieser Satzung erfolgt.

> 7. Alle Vorstandsmitglieder müssen zum Zeitpunkt des Amtsantritts:
>     1. das fünfundzwanzigste Lebensjahr nicht vollendet haben.
>     2. das siebzehnte Lebensjahr vollendet habe.

Reviewer 2: ```suggestion
7. Das Amt des Vorstandmitglieds darf nur antreten wer:
    1. das fünfundzwanzigste Lebensjahr nicht vollendet hat.
    2. das siebzehnte Lebensjahr vollendet hat.
```

Author: Ich sehe keine Veränderung der Aussage.

1. Die Amtsdauer der Vorstandsmitglieder beträgt zwei Jahre. Wiederwahl ist zulässig. 
2. Der Vorstand bleibt bis zur Neuwahl im Amt, damit auch nach Ablauf der Amtsdauer eine ordnungsgemäße gesetzliche Vertretung gesichert ist.
3. Der Vorstand besteht aus natürlichen Personen, den Vorstandsmitgliedern. Die Bekleidung mehrer Ämter durch eine Person ist zulässig, sofern dies auf Beschluss der Mitgliederversammlung erfolgt.
4. Jedes Vorstandsmitglied besitzt in Abstimmungen und Beschlussfassungen des Vorstands eine Stimme, unbeschadet der Anzahl der ausgeübten Ämter.
5. Scheidet ein Vorstandsmitglied während seiner Amtszeit aus, wählt der verbleibende Vorstand übergangsweise bis zur nächsten Mitgliederversammlung einen Ersatz. Sind zwei oder mehr der durch die Mitgliederversammlung gewählten Vorstandsmitglieder dauerhaft an der Ausübung ihres Amtes gehindert, so sind unverzüglich Nachwahlen anzuberaumen.
6. Die Abwahl eines Vorstandsmitglieds ist nur möglich, sofern zeitgleich eine Neubesetzung des freigewordenen Amtes gemäß dieser Satzung erfolgt.
7. Alle Vorstandsmitglieder müssen zum Zeitpunkt des Amtsantritts:
    1. das fünfundzwanzigste Lebensjahr nicht vollendet haben.

>     2. das siebzehnte Lebensjahr vollendet habe.

Author: ```suggestion
    2. das achzehnte Lebensjahr vollendet haben.
```
